pages/head/public.inc

```diff
@@ -1,5 +1,4 @@
 <meta http-equiv="Cache-control" content="No-Cache">
-<script src="sitesupport/common.js"></script>
```

Comment:
Not needed?

Reply:
We already include common.js in the top-level head-before that every single page gets! We've got a lot of redundancies like these that I'm picking off along the way.